src/components/opoTable/opoTable.js

```diff
@@ -14,88 +16,137 @@ export default function OpoTable({
         <thead>
-          <tr>
-            {inState ? (
-              <>
-                <Heading title="name" />
-                <Heading title="region" />
-              </>
-            ) : (
-              <>
-                <Heading title="states" />
-                <Heading title="name" />
-              </>
-            )}
-            <Heading title="tier" />
-            <Heading title="donors" />
-            <Heading title="shadow" color="red" />
-            {inState && <Heading title="investigation" />}
-          </tr>
+          {headerGroups.map(headerGroup => (
+            <tr {...headerGroup.getHeaderGroupProps()}>
+              {headerGroup.headers.map(column => (
+                <th
+                  {...column.getHeaderProps(column.getSortByToggleProps())}
+                  scope="col"
+                  className={column.color ?? null}
+                >
+                  <div className={styles.header}>
+                    {column.render("Header")}
+                    <div className={styles.chevron}>
+                      {column.isSorted ? (
+                        column.isSortedDesc ? (
+                          <ChevronUp />
+                        ) : (
+                          <ChevronDown />
+                        )
+                      ) : (
+                        <ChevronDownGrey />
+                      )}
+                    </div>
+                  </div>
+                </th>
+              ))}
+            </tr>
+          ))}
         </thead>
-        <tbody>
-          {opos.map(
-            ({
-              donors,
-              investigation,
-              name,
-              region,
-              shadows,
-              states,
-              tier,
-            }) => (
-              <tr key={name}>
-                {inState ? (
-                  <>
-                    <td>{name}</td>
-                    <td>{region}</td>
-                  </>
-                ) : (
-                  <>
-                    <td>{states}</td>
-                    <td>{name}</td>
-                  </>
-                )}
-                <td>
-                  <Container>
-                    <Tier className={styles.tierCol} tier={tier} />
-                  </Container>
-                </td>
-                <td className="text-center">{formatNumber(donors)}</td>
-                <td className={styles.shadows}>{formatNumber(shadows)}</td>
-                {inState && (
-                  <td className="text-center">
-                    {investigation ? "Yes" : "--"}
-                  </td>
-                )}
+        <tbody {...getTableBodyProps()}>
+          {rows.map(row => {
+            prepareRow(row);
+            return (
+              <tr {...row.getRowProps()}>
+                {row.cells.map(cell => {
+                  return (
+                    <td
+                      {...cell.getCellProps()}
+                      className={cell.column.cellClass ?? ""}
```

Comment:
nit: `?? null` for consistency with [the earlier change](https://github.com/Bloom-Works/opo-dashboard/pull/44/files#diff-6aeb201afe788e9bcc6826425ffbe754446112f40d0befe6b7624d599b5029e7R107)